src/main/java/rx/Completable.java

```diff
@@ -1843,7 +1846,7 @@ public final Completable startWith(Completable other) {
      */
     public final Subscription subscribe() {
         final MultipleAssignmentSubscription mad = new MultipleAssignmentSubscription();
-        subscribe(new CompletableSubscriber() {
+        unsafeSubscribe(new CompletableSubscriber() {
```

Comment:
Right. However, this method doesn't run any user supplied code and thus no need to wrap it into a safe completable subscriber.